### ISSUE TITLE
[2.6] Plant images folder remains in DCIM/GrowTracker after the plant is deleted

### ISSUE THREAD

**Issue by 9YbQiuEohUu1** [7LPdWcaW/GrowTracker-Android#162]
When deleting the plant, all of its images are deleted (as they should) but the empty folder remains in place. Since the folder name is a string of random characters, this will makes file operations a pain.

**Comment by 9YbQiuEohUu1** [comment]
Just tested in 2.6.1 and the issue is still present. Created 3 plants, added 1 image to each, and deleted all three. As before, the images are gone but the folders are not.

**Comment by 7LPdWcaW** [comment]
@9YbQiuEohUu1 was this when deleting the plant? or the images?

**Comment by 9YbQiuEohUu1** [comment]
@7LPdWcaW The plants.

**Comment by 9YbQiuEohUu1** [comment]
Okay, here's more details on the issue:
When deleting the plant, all its images are deleted but the folder (with the .nomedia file) isn't. When deleting all images from the plant's "Images" section, the folder is deleted as well.
Shouldn't it be the other way around? If the plant's entry is still there, there's still a possibility that other images will be added later (so the folder is still needed).

**Comment by 9YbQiuEohUu1** [comment]
> When deleting the plant, all its images are deleted but the folder (with the .nomedia file) isn't. When deleting all images from the plant's "Images" section, the folder is deleted as well.
> Shouldn't it be the other way around?

Will this be fixed before the 3.0 release? Looks like it hasn't been so far

**Comment by 7LPdWcaW** [comment]
Yes, all small bugs/changes will be done before 3.0 as 3.0 is a long term project with no release date set.
